cli/command/swarm/init.go

```diff
@@ -52,6 +53,8 @@ func newInitCommand(dockerCli command.Cli) *cobra.Command {
 	flags.SetAnnotation(flagDefaultAddrPool, "version", []string{"1.39"})
 	flags.Uint32Var(&opts.DefaultAddrPoolMaskLength, flagDefaultAddrPoolMaskLength, 24, "default address pool subnet mask length")
 	flags.SetAnnotation(flagDefaultAddrPoolMaskLength, "version", []string{"1.39"})
+	flags.Uint32Var(&opts.dataPathPort, flagDataPathPort, 4789, "Data Path port number")
```

Comment:
I _think_ we should remove the default here, as the default is set by the daemon.

Can you also move the new lines up (below the `data-path-addr` flag?

The flag description could be a bit more consistent with the existing flag, and (I think) should mention the valid port ranges;

```suggestion
	flags.Uint32Var(&opts.dataPathPort, flagDataPathPort, 0, "Port number to use for data path traffic (1024 - 49151). If no value is set or is set to 0, the default port (4789) is used.")
```